epilogue-processor/src/main/java/edu/wpi/first/epilogue/processor/EpilogueGenerator.java

```diff
@@ -152,13 +156,17 @@ public static boolean shouldLog(Logged.Importance importance) {
             out.println("  public static void bind(" + robotClassName + " robot) {");
             out.println("    robot.addPeriodic(() -> {");
             out.println("      long start = System.nanoTime();");
+            out.println("      config.loggingPeriod = Milliseconds.of(robot.getPeriod());");
+            out.println(
+                "      config.loggingPeriodOffset = Milliseconds.of(robot.getPeriod() / 2);");
```

Comment:
This doesn't seem right. This will overwrite the user-configured logging period after logging starts, but logging will be done at the _original_ configured period and offset at the time `bind` is called in the robot constructor

Reply:
Yeah I need to add a null check